doc/conf.py

```diff
@@ -0,0 +1,183 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+import os
+import sys
+from datetime import date
+
+# -- Path setup --------------------------------------------------------------
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Viziphant'
+authors = 'Elephant team'
+copyright = u"2019-{this_year}, {authors}".format(this_year=date.today().year,
```

Comment:
```suggestion
copyright = u"2017-{this_year}, {authors}".format(this_year=date.today().year,
```

Reply:
shouldn't it be 2015, as in the license?